compiler/one-cmds/onecc.template.cfg

```diff
@@ -1,29 +1,102 @@
+; # make `one-cmd` True and then uncomment its section
 [onecc]
-one-import-tf=True
+; # nn model to circle
+one-import-tf=False
 one-import-tflite=False
 one-import-bcq=False
 one-import-onnx=False
-one-optimize=True
+; # circle to circle with opt
+one-optimize=False
+; # circle to circle with quantization
 one-quantize=False
+; # partition circle
 one-partition=False
-one-pack=True
+; # nnpackage
+one-pack=False
+; # codegen from circle to backend's type
 one-codegen=False
+; # profile
 one-profile=False
 
-[one-import-tf]
-input_path=/path/to/inception_v3.pb
-output_path=inception_v3.circle
-input_arrays=input
-input_shapes=1,299,299,3
-output_arrays=InceptionV3/Predictions/Reshape_1
-converter_version=v1
-model_format=graph_def
-
-[one-optimize]
-input_path=inception_v3.circle
-output_path=inception_v3.opt.circle
-generate_profile_data=False
-
-[one-pack]
-input_path=inception_v3.opt.circle
-output_path=inception_v3_pack
+; [one-import-tf]
+; # mandatory
+; input_path=             ; pb file
+; output_path=            ; circle file
+; input_arrays=           ; input tensor name or names
+; input_shapes=           ; eg) 1,299,299,3
+; output_arrays=          ; output tensor name or names
```

Comment:
FYI, these are needed for v1. v2 don't need these.

Reply:
Very good point! I add `optional but mandatory for v1`